lib/config.js

```diff
@@ -1,9 +1,14 @@
+var service_host = process.env.MONGO_SERVICE_HOST
+
 var database = {
-    url: 'mongodb://localhost:27017/pacman',
+    url: `mongodb://${service_host}:27017/pacman`,
```

Comment:
I've switched this to use the MONGO_SERVICE_HOST env variable to remove the HostPort requirement.